runelite-client/src/main/java/net/runelite/client/plugins/screenshot/ScreenshotPlugin.java

```diff
@@ -351,7 +351,15 @@ public void onChatMessage(ChatMessage event)
 
 		if (config.screenshotKills() && KILL_MESSAGES.stream().anyMatch(chatMessage::contains))
 		{
-			String fileName = "Kill " + format(new Date());
+			String kill_name = chatMessage;
```

Comment:
Things are camelCased in Java.

Reply:
Yikes, too much Ruby, sorry. Will change. :)